src/main/java/org/jabref/gui/externalfiles/DownloadFullTextAction.java

```diff
@@ -112,9 +112,9 @@ private void downloadFullTexts(Map<BibEntry, Optional<URL>> downloads, BibDataba
                 Optional<Path> dir = databaseContext.getFirstExistingFileDir(Globals.prefs.getFilePreferences());
                 if (dir.isEmpty()) {
                     dialogService.showErrorDialogAndWait(Localization.lang("Directory not found"),
-                            Localization.lang("Main file directory not set!") + " "
-                                    + Localization.lang("Preferences")
-                                    + " -> " + Localization.lang("File"));
+                            Localization.lang("Main file directory not set!\n")
+                                    + Localization.lang("Check the Preferences (Linked files)")
+                                    + Localization.lang(" or the Library properties."));
```

Comment:
It's better and easier for the translators if you put this in one string. See https://devdocs.jabref.org/getting-into-the-code/code-howtos#using-localization-correctly

Reply:
We should also take care about casing - maybe less is better?

    Main file directory not set. Check the preferences (linked files) or the library properties.

Reply:
Great! I'll work on it

Reply:
Changed to one string and also changed the message 😄 
<img width="517" alt="Screen Shot 2020-12-28 at 10 55 33 AM" src="https://user-images.githubusercontent.com/33851952/103233913-83ca5a80-48fb-11eb-91b4-9644a3e06465.png">

Reply:
Thanks! Now you need to add this to the resources/l10n/en.properties file as well (see the LocalizationConsistencyTest) under l10n.
When the key contains spaces, you need to add a backslash after the word before the space. (see the other entries in the file)

Reply:
Now the test fail complaining that a newline is illegal - see: https://github.com/JabRef/jabref/pull/7232/checks?check_run_id=1618423949#step:6:122

```
java.lang.RuntimeException: Main file directory not set.nCheck the preferences (linked files) or the library properties. contains a new line character. As this is a localization key, this is illegal!
```

You can try with `<br>` or `<p>` in the localization - such as other localizations did. Could you please try and report back?

Reply:
BTW: Hint: How to read error outputs.

1. See the red crosses  
   ![grafik](https://user-images.githubusercontent.com/1366654/103300062-44d0fd80-49fe-11eb-9666-a2d2992bb69c.png)
2. Experience in JabRef tells: Click at Details at "Tests / Unit Tests"
3. Experience tells to scroll up, until one sees a red "Error"  
   ![grafik](https://user-images.githubusercontent.com/1366654/103300104-67631680-49fe-11eb-9c86-82a8d977501e.png)

Then try to understand the error message. I tried to explain the error message at my last commit. My wish, however, is that I am not the only one knowing how to click on "Details", scroll up and find "Error"

Reply:
> You can try with <br> or <p> in the localization - such as other localizations did. Could you please try and report back?

javafx no longer uses html, so \n is correct

Reply:
@Siedlerchr May I ask whether you've seent he screen shot error output? Here the deep link to the method throwing the exception: https://github.com/JabRef/jabref/blob/afdb1942c5ec2126e69c92c7f2660459a7bdbeae/src/test/java/org/jabref/logic/l10n/LocalizationParser.java#L287

It throws an exception if `\n` is contained there. Thus, we cannot use `\n`. Since you say that `<p>` and `<br>` cannot be used, we cannot have any newlines any more. Am I right?